dir/path.go

```diff
@@ -57,8 +57,14 @@ const (
 	PathConfigFile = "config.json"
 	// PathSigningKeys is the signingkeys file relative path.
 	PathSigningKeys = "signingkeys.json"
-	// PathTrustPolicy is the trust policy file relative path.
-	PathTrustPolicy = "trustpolicy.json"
+	// PathTrustPolicy is the OCI trust policy file relative path.
+	// Deprecated: PathTrustPolicy exists for historical compatibility and should not be used.
+	// To get OCI trust policy path, use PathOCITrustPolicy.
+	PathTrustPolicy = PathOCITrustPolicy
+	// PathOCITrustPolicy is the OCI trust policy file relative path.
+	PathOCITrustPolicy = "trustpolicy.json"
```

Comment:
According to our spec: https://github.com/notaryproject/specifications/blob/main/specs/trust-store-trust-policy.md#trust-policy,  `PathOCITrustPolicy` should have two valid values, they are `trustpolicy.json` and `trustpolicy.oci.json`.

Reply:
Yep, added a fallback logic